include/RE/B/BGSMod.h

```diff
@@ -45,72 +48,114 @@ namespace RE::BGSMod
+				{
+					struct ValueType1Bool
+					{
+						bool  value1;
+						bool  value2;
+						void* pad;
+					};
+
+					struct ValueType2Float
+					{
+						float value1;
+						float value2;
+						void* pad;
+					};
+
+					struct ValueType3Int
+					{
+						std::int32_t value1;
+						std::int32_t value2;
+						void* pad;
+					};
+
+					struct ValueType4FormFloat
+					{
+						RE::TESForm* value1;
+						float        value2;
+					};
+
+					union // 0
+					{
+						ValueType1Bool      type1Val;
+						ValueType2Float     type2Val;
+						ValueType3Int       type3Val;
+						ValueType4FormFloat type4Val;
+					};
+
+					std::uint8_t  type;       // 10
+					std::uint8_t  pad[7];     // 11
+					std::uint32_t propertyID; // 18
+					std::uint32_t pad2;       // 18
+					// std::uint64_t pad2;
+					// float         pad2;
+					// float Value1;
+					// float Value2;
+				};
+
+				static_assert(sizeof(Property) == 0x20);
+				static_assert(offsetof(Property, type) == 0x10);
+
+				BSTArray<Include>  includes;   // 00
+				BSTArray<Property> properties; // 10
 			};
+
 			static_assert(sizeof(Data) == 0x20);
 
-			~Mod() override;  // 00
+			~Mod() override; // 00
 
 			// members
-			Data                 data;           // 80
-			BGSAttachParentArray attachParents;  // A0
-			std::uint64_t        unkC0;          // C0 - std::vector<BGSKeyword*>?
-			std::uint64_t        unkC8;          // C8
-			std::uint64_t        unkD0;          // D0
-			BGSKeyword*          attachPoint;    // D8
-			BGSMorphableObject*  unkE0;          // E0
-			BGSMorphableObject*  unkE8;          // E8
-			BSFixedString        name;           // F0
-			std::uint8_t         unkF8;          // F8
-			std::uint8_t         unkF9;          // F9
-			std::uint8_t         priority;       // FA
-			std::uint8_t         unkFB;          // FB
-			std::uint8_t         unkFC;          // FC
+			Data                 data;          // 80
+			BGSAttachParentArray attachParents; // A0
+			std::uint64_t        unkC0;         // C0 - std::vector<BGSKeyword*>?
+			std::uint64_t        unkC8;         // C8
+			std::uint64_t        unkD0;         // D0
+			BGSKeyword* attachPoint;   // D8
+			BGSMorphableObject* unkE0;         // E0
+			BGSMorphableObject* unkE8;         // E8
+			BSFixedString        name;          // F0
+			std::uint8_t         unkF8;         // F8
+			std::uint8_t         unkF9;         // F9
+			std::uint8_t         priority;      // FA
+			std::uint8_t         unkFB;         // FB
+			std::uint8_t         unkFC;         // FC
 		};
+
 		static_assert(sizeof(Mod) == 0x108);
-	}
+	} // namespace Attachment
 
 	namespace Template
 	{
-		class __declspec(novtable) Item :
-			public TESFullName,       // 00
-			public BGSMod::Container  // 10
-		{
-		public:
-			SF_RTTI_VTABLE(BGSMod__Template__Item);
-
-			~Item() override;  // 00
-		};
-
-		class __declspec(novtable) Items :
-			public BaseFormComponent  // 00
+		class Items : public BaseFormComponent
```

Comment:
Is your fork out of date, or did you remove these for a reason?

Reply:
Could be, yes. It's my very first time trying this ^^ I wanted to contribute at least something